### DISCUSSION TITLE
posts/Spring-Boot-Test-Service-%ED%85%8C%EC%8A%A4%ED%8A%B8/

### DISCUSSION THREAD

**Discussion by giscus** [zhyunk/zhyunk#3] [General]
# posts/Spring-Boot-Test-Service-%ED%85%8C%EC%8A%A4%ED%8A%B8/

JUnit 5, Mockito와 함께하는 Service 코드 테스트!

https://blog.zhyun.kim/posts/Spring-Boot-Test-Service-%ED%85%8C%EC%8A%A4%ED%8A%B8/

**Comment by zhyunk** [comment]
<kbd>추가</kbd> **@​Spy** 사용과 관련해서 참고하면 좋을것 같은 블로그 포스팅: https://mangkyu.tistory.com/159
